endpoints/openrtb2/video_auction.go

```diff
@@ -242,7 +287,13 @@ func cleanupVideoBidRequest(videoReq *openrtb_ext.BidRequestVideo, podErrors []P
 	return videoReq
 }
 
-func handleError(labels *pbsmetrics.Labels, w http.ResponseWriter, errL []error, vo *analytics.VideoObject) {
+func handleError(labels *pbsmetrics.Labels, w http.ResponseWriter, errL []error, vo *analytics.VideoObject, debugLog *exchange.DebugLog) {
+	if debugLog != nil && debugLog.EnableDebug {
+		if rawUUID, _ := uuid.NewV4(); len(rawUUID) > 0 {
```

Comment:
This will produce warnings that you're ignoring the error from NewV4. In practice, you're likely fine with a check on the uuid.. but checking for a non-nil error is preferred.

Reply:
Updated to check the error instead